app/views/external_users/admin/external_users/index.html.haml

```diff
@@ -10,32 +10,34 @@
 = render partial: 'shared/search_form', locals: { search_path: external_users_admin_external_users_path(anchor: 'search-button'), hint_text: t('hint.search_users'), button_text: t('search.users') }
 
 
-= govuk_table_and_caption t('.table_caption', provider_name: "#{current_user.provider.name}"), class: 'govuk-visually-hidden' do
+= govuk_table(class: 'report') do
+  = govuk_table_caption(class: 'govuk-visually-hidden') do
+    = t('.table_caption', provider_name: "#{current_user.provider.name}")
 
-  = govuk_table_thead_collection [t('.last_name'),
+  = govuk_table_thead_collection [ t('.last_name'),
   t('.first_name'),
   t('.supplier_number'),
   t('.email'),
   t('.email_confirmation'),
-  t('.actions')]
+  t('.actions') ]
 
   = govuk_table_tbody do
     - @external_users.each do |advocate|
       = govuk_table_row do
-        = govuk_table_td advocate.last_name
+        = govuk_table_td( 'data-label': t('.last_name') ) { advocate.last_name }
 
-        = govuk_table_td advocate.first_name
+        = govuk_table_td( 'data-label': t('.first_name') ) { advocate.first_name }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4166/commits/b5768c8a8d9f832e8b017cb4bf958381030622e6#r683550696)